### PR TITLE
Fix sdpa_micro which caused accuracy issue of long-prompt wwb

### PR DESCRIPTION
**Pull request by byungilm**
### Description of the issue
 - WWB accuracy test with long prompts showed low similarity on multiple platforms.
long-wwb

#### The code and line that caused this issue
 - When WWB is executed by MIXED stagemode on the target platforms, sdpa config is generated by choose_config_xe2 in sdpa_ge_micro.cpp. 
  This refers to invalid values of 'wg_n_kq' and 'wg_n_vs' which are pre-defined.

#### Reproduction step and snapshot
 - Reproduced by benchmark on dg2-512 or on MTL
 `python benchmark.py -t text_gen -d GPU.1 -m llama-3.2-3b-instruct/pytorch/ov/OV_FP16-INT8_ASYM -pf long-prompt.jsonl --cb_config "{\"enable_prefix_caching\":true}"`
 - cb-config option is required to enable MIXED stage-mode.
 - sample long prompt is attached to the ticket 
[long-prompt.txt](https://github.com/user-attachments/files/25690220/long-prompt.txt)

#### Checklist
 - [x] Is it a proper fix?
 - [x] Did you include test case for this fix, if necessary? 
 - [x] Did you review existing test that can be extended to cover this scenario? Passed llm_bench
     
### Tickets:
 - CVS-181748, CVS-181751, CVS-181752

### AI Assistance:
 - *AI assistance used: no

### CODE REVIEW

File: src/plugins/intel_gpu/tests/unit/test_cases/paged_attention_gpu_test.cpp [hunk 1]
```diff
@@ -1947,6 +1947,9 @@ INSTANTIATE_TEST_SUITE_P(smoke_paged_attention, paged_attention_test, ::testing:
...
     paged_attention_test_params{ {{1008, 792}}, 32, 32, 128, 128, 16, {100.0}, 256, false, ENABLE_CACHE_COMPRESSION,ov::internal::CacheQuantMode::BY_TOKEN, STATIC_INPUT_PAD, DISABLE_SCORES, PER_TOKEN_ROTATION, DISABLE_FA_V2 }, // mixed: prefix caching
     paged_attention_test_params{ {{1, 34}, {2, 20}, {10, 34}}, 2, 2, 64, 64, 16, {100.0}, 10, false, ENABLE_CACHE_COMPRESSION,ov::internal::CacheQuantMode::BY_TOKEN, STATIC_INPUT_PAD, ENABLE_SCORES, PER_TOKEN_ROTATION, DISABLE_FA_V2 }, // mixed: 2nd token + 1st token + part of 1st token
+    // mixed: 2nd token + 1st token + part of 1st token & head size 128
+    paged_attention_test_params{ {{1, 34}, {2, 20}}, 2, 2, 128, 128, 16, {100.0}, 10, false, ENABLE_CACHE_COMPRESSION,ov::internal::CacheQuantMode::BY_TOKEN, STATIC_INPUT_PAD, ENABLE_SCORES, DISABLE_ROTATION, DISABLE_FA_V2 },
+    paged_attention_test_params{ {{1, 34}, {2, 20}}, 2, 2, 128, 128, 16, {100.0}, 10, false, ENABLE_CACHE_COMPRESSION,ov::internal::CacheQuantMode::BY_CHANNEL, STATIC_INPUT_PAD, ENABLE_SCORES, DISABLE_ROTATION, DISABLE_FA_V2 },
```
Comment (e-ddykim) [inline src/plugins/intel_gpu/tests/unit/test_cases/paged_attention_gpu_test.cpp]: Please check if these are the issue case. From my local setup, `sdpa_micro` was not selected for these cases.

Comment (byungilm) [reply src/plugins/intel_gpu/tests/unit/test_cases/paged_attention_gpu_test.cpp]: I got analysis why the unit-tests did not reproduced.
wg_n_kq/wg_n_vs = 1 is a correctness bug where intra-workgroup reduction over KV blocks is insufficient, causing incorrect attention results for long contexts (~30+ blocks).

wg_n_kq/wg_n_vs = 1 reduces the number of subgroups within a workgroup that process KV blocks. This introduces accumulation errors in attention, but the error magnitude can fall within tolerance=2e-3 on fp16 random inputs. In other words, this bug is a correctness issue that cannot be detected by floating-point absolute-error-based unit tests. It is only amplified by the peaked softmax distributions produced by real model weights.

Comment (e-ddykim) [reply src/plugins/intel_gpu/tests/unit/test_cases/paged_attention_gpu_test.cpp]: I don't understand what this means. `wg_n_kq/wg_n_vs` is still 1. Anyway, let's remove the test cases if it's not easy to reproduce the issue case.
